includes/shortcodes/class-sensei-legacy-shortcodes.php

```diff
@@ -235,6 +235,7 @@ public static function user_courses( $atts, $content = null ) {
 
 		if ( is_user_logged_in() ) {
 
+			wp_enqueue_script( Sensei()->token . '-user-dashboard' );
```

Comment:
We should check for the `js_disable` setting here as well. Otherwise it's possible we'll enqueue a script that was never registered.